standalone-e2e-test/src/test/java/response/Links.java

```diff
@@ -0,0 +1,6 @@
+package response;
+
```

Comment:
POJOs are usually encapsulated, meaning that their field are private and can be accessed only by setters and getters.